app/config/config.yml

```diff
@@ -72,6 +72,7 @@ framework:
         engines:
           - mustache
           - twig
+    web_link: ~
```

Comment:
What does this do?

Reply:
Enables the Symfony component that adds the link header to the response.

Took me a little while to realise the Twig extension is always enabled but it won’t do anything without this. Will open an upstream PR.

Reply:
Opened https://github.com/symfony/symfony/pull/27626.